tests/helics/CMakeLists.txt

```diff
@@ -46,4 +47,4 @@ add_test(find-package-tests ${CMAKE_CTEST_COMMAND} -C ${CMAKE_BUILD_TYPE}
                             "-DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}"
                             "-DCMAKE_CXX_STANDARD=${CMAKE_CXX_STANDARD}"
 )
-set_property(TEST find-package-tests PROPERTY LABELS Continuous Daily)
+set_property(TEST find-package-tests PROPERTY LABELS Packaging)
```

Comment:
Will appveyor still run the packaging tests?

Reply:
If not it probably should

Reply:
Okay, it’s running those test on appveyor.